Data_Wrangling/README.md

```diff
@@ -0,0 +1,15 @@
+I preprocessed a dataset of over 500,000 rows, extracting features and cleaning the data through the following major steps:
```

Comment:
No need to use first person while handling a README. I'll let it pass for this PR, but next time, avoid that.